code/modules/events/immovable_rod/immovable_rod.dm

```diff
@@ -247,10 +247,16 @@
 	strongman.client?.give_award(/datum/award/achievement/jobs/feat_of_strength, strongman)
 	strongman.visible_message(
 		span_boldwarning("[strongman] suplexes [src] into the ground!"),
-		span_warning("You suplex [src] into the ground!")
+		span_warning("As you suplex [src] into the ground, your body ripples with power!")
```

Comment:
I think they will realize their athletics went up from the messages they get from it levelling up, is this extra text necessary?

Reply:
Maybe, if only because I think understanding why is flavorful.

It'd be arranged something like;
This text -
``You feel your Athletics improving!`` or something -
And then maybe any text you get from leveling up.